### DISCUSSION TITLE
How to use Curator Media Tool

### DISCUSSION THREAD

**Discussion by Jubeki** [awcodes/scribble#21] [Q&A] [answered]
I have seen that the "CuratorMedia" class exists, but I don't know how to get it working.

I tried registering it as a tool in the AppServiceProvider, but I probably also need to do something else, like creating a custom profile right?

```php
app(ScribbleManager::class)
    ->registerTools([
        CuratorMedia::make(),
    ]);
```                

Could somebody please provide some pointers or Code snippets?

**Comment by awcodes** [comment, marked as answer]
You will also need to create a custom profile to use instead of the default one so you can replace the 'media' with 'curator-media' in the appropriate toolbar, bubble or suggestion tools.
